src/components/footer/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import logo from "./assets/ac_logo_24.png";
+import BottomNavigation from '@material-ui/core/BottomNavigation';
+import BottomNavigationAction from '@material-ui/core/BottomNavigationAction';
+import styles from "./footer.module.scss";
+
+
+export default function Footer() {
+
+
+  return (
+    <BottomNavigation className={styles.Footer} showLabels>
+      <BottomNavigationAction href="/" alt="Above Curve Homepage" icon={<img src={logo} alt="Above Curve" data-testid="ac-logo"></img>} />
```

Comment:
Might want to change this to be `data-testid="ac-footer-logo"` (and in the unit tests too)